Cleanup EXIREdgeDialectVerifier to use the same logic to verify core

larryliu0820 opened the pull request:
Summary:
Making sure EXIREdgeDialectVerifier is able to verify core ATen ops
correctly. Getting rid of forked logic of `check_valid_aten_op`.

Differential Revision: D52492252

iseeyuan reviewed (approved):
LGTM.

Not for this diff, but in the future is the verifier going to support ops reserved not for decomposition?

larryliu0820 commented:
> LGTM.
> 
> Not for this diff, but in the future is the verifier going to support ops reserved not for decomposition?

Yeah if we can specify what ops to be reserved from decomposition through config, the same config can be used for verifier as well.

larryliu0820 committed:
Cleanup EXIREdgeDialectVerifier to use the same logic to verify core (#1504)

Summary:

Making sure EXIREdgeDialectVerifier is able to verify core ATen ops
correctly. Getting rid of forked logic of `check_valid_aten_op`.

Reviewed By: iseeyuan

Differential Revision: D52492252
